.gitignore

```diff
@@ -1,3 +1,9 @@
 /vendor/
 build/
 composer.lock
+composer.phar
```

Comment:
Can we please not use them in `.gitignore`. They belong to `~/.gitignore` file, not ours.